nilearn/image/tests/test_resampling.py

```diff
@@ -906,7 +967,7 @@ def test_resampling_with_int_types_no_crash(dtype):
 @pytest.mark.parametrize("dtype", ["int64", "uint64", "<i8", ">i8"])
 @pytest.mark.parametrize("no_int64_nifti", ["allow for this test"])
```

Comment:
This parametrization seem ununsed

Reply:
ah it is a fixture from conftest.py

Reply:
I believe it is. IIUC it disables preventing creating int64 data. See https://github.com/nilearn/nilearn/blob/0acf94b92f262ceb01eafacd5f9b51adf1f21bf5/nilearn/conftest.py#L30-L68